### PR TITLE
Node ci

### PR DESCRIPTION
**Pull request by 8ball030**

### CODE REVIEW

File: frontend/src/pages/bot_management/AgentList.js [hunk 1]
```diff
@@ -94,8 +87,8 @@ class AgentList extends React.Component {
                   </TableCell>
                   <TableCell align="right">{agentsData[row].date_created}</TableCell>
                   <TableCell align="right">{agentsData[row].date_updated}</TableCell>
-                  <TableCell align="right">{agentsData[row].eth_val}</TableCell>
-                  <TableCell align="right">{agentsData[row].usd_val}</TableCell>
+                  <TableCell align="right">{agentsData[row].eth_val/ 100000000}</TableCell>
+                  <TableCell align="right">{agentsData[row].usd_val/ 100000000}</TableCell>
```
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply frontend/src/pages/bot_management/AgentList.js:91]: Then calculations like these need be wrapped in tested helper modules so we make sure we know what we're doing and also simplifies reading the code.

**Review by DavidMinarsch — APPROVED** [review]
LGTM, some comments

File: agents/autonomous_hegician/skills/option_management/web_server.py [hunk 2]
```diff
@@ -189,15 +188,9 @@ def aggregate(row, ret):
...
-                int(Web3.fromWei(row.breakeven, "ether")) if row.breakeven != 0 else 0
-            )
-            ret["current_pnl"] = (
-                int(Web3.fromWei(row.current_pnl, "ether"))
-                if row.current_pnl != 0
-                else 0
-            )
+            ret["amount"] = int(row.amount)
+            ret["breakeven"] = int(row.breakeven)
+            ret["current_pnl"] = int(row.current_pnl)
```
[the remark this answers is not in the document]
Comment (8ball030) [reply agents/autonomous_hegician/skills/option_management/web_server.py:193]: For reference, all denominations are in WEI

[the remark this answers is not in the document]
Comment (8ball030) [reply agents/autonomous_hegician/skills/option_management/web_server.py:193]: (will add to docs on next pull)

File: frontend/src/pages/bot_management/AgentList.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (8ball030) [reply frontend/src/pages/bot_management/AgentList.js:91]: Would like to catch up about this, I know its non optimal the way ime doing ti here, however I am not how we could achieve that and keep the front end responsiveness..

**Pull request merged**